packages/Autocomplete/src/Autocomplete.tsx

```diff
@@ -76,6 +79,9 @@ Autocomplete.defaultProps = {
   selected: (): void => {
     return;
   },
+  onChange: (): void => {
```

Comment:
Do we need a default as you have a check on line 32 and if we set default it will always be truthy?